tests/test_models_dir_flag.py

```
@@ -0,0 +1,179 @@
+
+@pytest.mark.pre_merge
+@pytest.mark.unit
+@pytest.mark.gpu_0
+def test_models_dir_nonexistent_exits_with_code_2(tmp_path):
+    missing = tmp_path / "no_such_dir"
+    # Run from the project root so conftest.py is discovered and --models-dir
+    # is registered before pytest_configure fires.
+    # Note: the child pytest process collects from this file itself — keep
+    # module-level imports here side-effect-free to avoid spurious child failures.
+    project_root = Path(__file__).parents[1]
+    result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)